### PR TITLE
8738 Changed Coop Type to a key

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#8738

*Description of changes:*
- updated Cooperative Type v-select items
- converted coop type on summary page
- changed Coop Type enum to type codes
- added helper to convert coop type to description
- added enum mixin unit test
- app version: 2.0.18

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

File: src/resources/cooperative.ts [hunk 1]
```diff
@@ -80,8 +80,8 @@ export const CooperativeResource: ResourceIF = {
         section2: {
           label: 'The Rules need to:',
           items: [
-            { type: ItemTypes.TEXT, value: `Balance the rights of individual members with the interests of the Cooperative
-              Association as a whole` },
+            { type: ItemTypes.TEXT, value: `Balance the rights of individual members with the interests of the
+              Cooperative Association as a whole` },
```
Comment (severinbeauvais) [inline src/resources/cooperative.ts:84]: Fixed lint issue (line was too long).

File: src/store/getters/state-getters.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { AccountTypes, CoopType, CorpTypeCd, RoleTypes } from '@/enums'
+import { AccountTypes, CoopType, CorpTypeCd } from '@/enums'
```
Comment (severinbeauvais) [inline src/store/getters/state-getters.ts:1]: RoleTypes was not used.

File: src/store/getters/state-getters.ts [hunk 3]
```diff
@@ -191,11 +191,6 @@ export const getNameTranslations = (state: StateIF): NameTranslationIF[] => {
   return state.stateModel.nameTranslations
 }
 
-/** The Office Addresses object. */
-export const getOfficeAddresses = (state: StateIF): IncorporationAddressIF | {} => {
```
Comment (severinbeauvais) [inline src/store/getters/state-getters.ts:195]: getOfficeAddresses() was not used.

**Review by eve-git — APPROVED** [review]
LGTM

**Pull request merged**